es/models.tex

```diff
@@ -164,11 +165,11 @@ \chapter{Modelos mentales y evaluación formativa}\label{s:models}
 (de manera que pueda ser usada en grupos).\index{formative assessment!requirements}
 El tipo de evaluación formativa más ampliamente usado es probablemente
 el cuestionario de opciones múltiples (COM).\index{multiple choice question}
```

Comment:
Estaba justo pasando el capítulo de In the Classroom y allí también se mencionan los cuestionarios de opciones múltiples, pero como preguntas de opción múltiple y removí toda mención a la sigla porque no es conocida y por ende dificultaba la lectura.
Deberíamos decir si usamos:
- cuestionario de opción múltiple
- preguntas de opciones múltiples 
- pregunta de opción múltiple

y si usamos o no una sigla. Es este el primer capítulo en el que aparece, verdad?

Reply:
Coincido en sacar la sigla!!
Sí, este capítulo es la primera aparición del término. Lo había anotado para comentarlo en glossary al final de todo.
Para mí es: 
- si hay 1 pregunta: pregunta de opción múltiple.
- si hay varias preguntas: preguntas de opciones múltiples; o cuestionario de opciones múltiples (cuestionario significa varias preguntas)

Reply:
Me gusta más preguntas en plural que cuestionario.  Vamos por: 

- pregunta de opción múltiple (si hay una pregunta)
- preguntas de opción múltiple (si hay varias preguntas)